### DISCUSSION TITLE
Collection of articles about FreshRSS

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#4236] [Q&A] [answered]
Hi,

I can remember that somewhere we had a link list of articles that wrote about FreshRSS but I cannot find it anymore.

Where is it?

**Comment by Frenzie** [comment]
Pinging @Alkarex, probably.

**Reply by Alkarex** [reply]
* https://www.nextinpact.com/article/28150/106286-intelligence-social-freshrss-veut-redonner-coup-fouet-aux-agregateurs-flux
* https://fr.wikipedia.org/wiki/FreshRSS (should be updated, best by someone else)
* http://alternativeto.net/software/freshrss/ (we should encourage votes)
* https://framalibre.org/content/freshrss
* https://www.linuxuserspace.show/217
* https://app.flus.fr/collections/1672365565618356108 (dynamic list maintained by @marienfressinaud )
* https://app.flus.fr/collections/1698850762083905588 (hosting services)

@marienfressinaud probably has some more, e.g. from social networks

**Reply by marienfressinaud** [reply]
Few days late, I see that @math-GH found the answer. I also RT tweets on Twitter sometimes: https://twitter.com/FreshRSS

**Comment by math-GH** [comment, marked as answer]
I found it :)

=> https://github.com/FreshRSS/freshrss.org/issues/9
